packages/client/src/types/routing.ts

```diff
@@ -46,7 +46,7 @@ export type Chain = {
   logoURI?: string;
   bech32Prefix: string;
   feeAssets: FeeAsset[];
-  chainType: ChainType;
+  chainType: ChainType | string;
```

Comment:
let's keep using enum
```suggestion
  chainType: ChainType;
```